### PR TITLE
[Legacy Maps Plugin] Prevent reverse-tabnabbing

### PR DESCRIPTION
**Pull request by AMoo-Miki**
Signed-off-by: Miki <miki@amazon.com>

### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [X] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/maps_legacy/public/map/map_messages.js [hunk 1]
```diff
@@ -112,7 +112,11 @@ export const createZoomWarningMsg = (function () {
               Please go to { wms } for more information."
               values={{
                 wms: (
-                  <a target="_blank" href="https://opensearch.org/docs/latest/dashboards/maptiles/">
+                  <a
+                    target="_blank"
+                    href="https://opensearch.org/docs/latest/dashboards/maptiles/"
+                    rel="noopener noreferrer"
+                  >
```
Comment (joshuarrrr) [inline src/plugins/maps_legacy/public/map/map_messages.js:119]: This fix is fine, but if the modal was rendered using standard OUI components (e.g. `<OuiLink>`) this wouldn't be necessary: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1709#pullrequestreview-1000457712

My preference is to opportunistically clean up and standardize these types of antipatterns in the code when reasonable.

**Review by joshuarrrr — APPROVED** [review]

File: src/plugins/maps_legacy/public/map/map_messages.js (hunk 1 above)
Comment (ashwin-pc) [reply src/plugins/maps_legacy/public/map/map_messages.js:119]: +1 @AMoo-Miki Cant we make this change right away?

**Review by kavilla — APPROVED** [review]
LGTM but I agree with Josh and Ashwin.

File: src/plugins/maps_legacy/public/map/map_messages.js (hunk 1 above)
Comment (AMoo-Miki) [reply src/plugins/maps_legacy/public/map/map_messages.js]: Done.

**Review by joshuarrrr — APPROVED** [review]

**Pull request merged**